examples/features/clustered/clustered-jgroups/src/main/resources/activemq/server1/broker.xml

```diff
@@ -42,9 +42,9 @@ under the License.
 
       <broadcast-groups>
          <broadcast-group name="my-broadcast-group">
-            <broadcast-period>5000</broadcast-period>
             <jgroups-file>test-jgroups-file_ping.xml</jgroups-file>
             <jgroups-channel>active_broadcast_channel</jgroups-channel>
+            <broadcast-period>5000</broadcast-period>
```

Comment:
This is an unnecessary change

Reply:
Will do. I was verifying the functionality works since it didn't appear there were lots of tests. 

To your knowledge is there a good place for the tests to live?

Reply:
Depends on the test, if its a very small unit within the module itself. But in general theres actually a tests module where tests are divided into type, which will run with different maven profiles.